src/Microsoft.AspNet.Mvc.ViewFeatures/Controller.cs

```diff
@@ -424,6 +424,33 @@ public virtual JsonResult Json(object data, [NotNull] JsonSerializerSettings ser
         }
 
         /// <summary>
+        /// Creates a <see cref="HttpOkResult"/> object that produces an empty OK (200) response.
+        /// </summary>
+        /// <returns>The created <see cref="HttpOkResult"/> for the response.</returns>
+        [NonAction]
+        public virtual HttpOkResult Ok()
```

Comment:
Yeah don't care about class names. Do care about compat mostly with MVC, but secondarily about Web API. So because MVC didn't have this one way or another, let's keep it with the Web API name, so `Ok` is :ok: per Ryan.